src/lib/theme/_overrides.scss

```diff
@@ -89,6 +89,28 @@ $shape-radius: 8px;
     }
   }
 
+  // Icons - Swap based on app theme
+  .light-theme {
```

Comment:
I would remove this wrapper class and have it be the default, since this class sometimes is not added.

Reply:
It's already that way based on the SVG internal CSS but yeah can't hurt